### PR TITLE
Added planning docs to planning_documents branch - this documents our…

### PR DESCRIPTION
**Pull request by DaveLawes**
… discussions on the first day

### CODE REVIEW

File: domain_model.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      |
+      |
+      |
+   app.rb                       Model
+╔════════════╗             ╔════════════╗
+║            ║------------>║            ║
+║ Controller ║             ║    User    ║
+║            ║<------------║   (class)  ║
+╚════════════╝             ╚════════════╝
+      |                           |
```
Comment (rebeccasedgwick) [inline domain_model.md]: Double arrows from Controller to Views

File: domain_model.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+║    RACK    ║ Rack provides the DSL to write the web app
+║            ║
+╚════════════╝
+      |
+      |
+      |
+   app.rb                       Model
+╔════════════╗             ╔════════════╗
+║            ║------------>║            ║
+║ Controller ║             ║    User    ║
```
Comment (rebeccasedgwick) [inline domain_model.md]: Rename User box to have just Model

File: domain_model.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      |                           |
+╔════════════╗                    |              ╔════════════╗
+║            ║                    |              ║            ║
+║   Views    ║                    |------------->║  Property  ║
+║            ║                    |              ║  (class)   ║
+╚════════════╝                    |              ╚════════════╝
+ Embedded Ruby?                   |                     |
+                                  |              ╔════════════╗    SQL query    ╔════════════╗
+                                  |              ║            ║---------------->║    SQL     ║
+                                  |------------->║ActiveRecord║ {result object} ║  Database  ║
```
Comment (rebeccasedgwick) [inline domain_model.md]: ActiveRecord box name to change to User Class.

ActiveRecord retrieves the database info & wraps them as useable objects so would make sense for it to be labelled on the route between the classes and the database. This means that both classes will need to have routes directly to the database.

File: domain_model.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+   app.rb                       Model
+╔════════════╗             ╔════════════╗
+║            ║------------>║            ║
+║ Controller ║             ║    User    ║
+║            ║<------------║   (class)  ║
+╚════════════╝             ╚════════════╝
+      |                           |
+╔════════════╗                    |              ╔════════════╗
+║            ║                    |              ║            ║
+║   Views    ║                    |------------->║  Property  ║
```
Comment (rebeccasedgwick) [inline domain_model.md]: Double arrows for info exchange to & from Controller <-> Views

File: domain_model.md (hunk 2 above)
Comment (rebeccasedgwick) [inline domain_model.md]: Rename User box to have just Model (in line with rest of structure, and shows routing more clearly)

Comment (DaveLawes) [reply domain_model.md]: Updated!

File: domain_model.md (hunk 4 above)
Comment (DaveLawes) [reply domain_model.md]: this is opening a can of worms, can i leave it for now?

**Pull request merged**